kbase-extension/static/kbase/js/widgets/narrative_core/kbaseCellToolbarMenu.js

```diff
@@ -622,8 +523,9 @@ define([
     }
 
     return {
-        make: function (config) {
-            return factory(config);
+        // eslint-disable-next-line no-unused-vars
```

Comment:
why not just remove the `_config`?

Reply:
Just making eslint happy, whilst preserving the original intent (to communicate that a config object is provided). Now removed.